app/src/main/java/com/example/internetshop/data/cache/TokenPreference.kt

```diff
@@ -16,6 +16,7 @@ class TokenPreference @Inject constructor(context: Context) {
         context.getSharedPreferences(PREFERENCE_NAME, Context.MODE_PRIVATE)
 
     fun getToken(): Token {
+        setToken(Token(null,Date()))
```

Comment:
What is it? The first line of getToken is setToken. Everything is correct here?

Reply:
fixed